<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="btnStudentinfo">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Student Info</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>C:/Users/NGR/Desktop/book.png</normaloff>C:/Users/NGR/Desktop/book.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSechedule">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Cource Sechedule</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/sechedule1.ico</normaloff>:/icon/sechedule1.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnFinance">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Finance</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/money-box.png</normaloff>:/icon/money-box.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnHonorWall">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Honor Wall</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/rewards.ico</normaloff>:/icon/rewards.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSystemset">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>System Setting</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/tool.png</normaloff>:/icon/tool.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>27</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="pageStudentinfo">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageSechedule">
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>140</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageHonor">
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageFinance">
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="pageSystemsetting">
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
